README.md

```diff
@@ -5,52 +5,63 @@
-4. In the file `vuplay-jwplayer.js` replace `<fairplay-cert-url>` with the URL to the fairplay certificate. Please contact support@vualto.com if you do not have this URL.
-5. In the file `vuplay-jwplayer.js` replace `<your-vudrm-token>` with a VUDRM token from [https://admin.vudrm.tech](https://admin.vudrm.tech)
+2. In the file `studiodrm-jwplayer.js` replace `<dash-stream-url>` with your [MPEG-DASH](https://en.wikipedia.org/wiki/Dynamic_Adaptive_Streaming_over_HTTP) stream URL.
+3. In the file `studiodrm-jwplayer.js` replace `<hls-stream-url>` with your [HLS](https://developer.apple.com/streaming/) stream URL.
+4. In the file `studiodrm-jwplayer.js` replace `<fairplay-cert-url>` with the URL to the fairplay certificate. Please contact <support@jwplayer.com> if you do not have this URL.
+#### TODO: Change the below (no.5) to the JW Studio DRM docs
+5. In the file `studiodrm-jwplayer.js` replace `<your-studiodrm-token>` with a Studio DRM token from [https://jw-studio-drm-token-docs](studiodrm-token-docs)
 6. In the file `index.html` replace `<jwplayer-key>` with your JWPlayer key.
 7. Run `npm run build` in the project's root. This will create a `dist` folder that contains all the files needed to run this demo. 
-8. Add the host `jwplayer.vuplay.local.drm.technology` to your local machine's hosts file.
-9. Load a supported browser and go to `https://jwplayer.vuplay.local.drm.technology:14703`
+8. Add the host `jwplayer.studiodrm.local` to your local machine's hosts file.
+9. Load a supported browser and go to `https://jwplayer.studiodrm.local:14703`
 
-You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact support@vualto.com or [JW Player](https://www.jwplayer.com/pricing/) for more information. 
+You will need an enterprise license with JW Player in order to playback DRM encrypted content. Please contact <support@jwplayer.com> or [JW Player](https://www.jwplayer.com/pricing/) for more information. 
 
-To retrieve the JW Player key [click here](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
+Retrieve the [JW Player key](https://dashboard.jwplayer.com/#/players/downloads) and copy the license key for the appropriate player in the `Downloads` section.
 
 In order to allow DRM encrypted playback in chrome (<https://goo.gl/EEhZqT>), SSL has been enabled for the demo. You will get a warning about an invalid cert `NET::ERR_CERT_AUTHORITY_INVALID` but this can safely be ignored.
 
 ### Browser support
 
 The browser must support [encrypted media extensions](https://www.w3.org/TR/2016/CR-encrypted-media-20160705/).
 Currently this includes the latest versions of Chrome, Firefox, Internet Explorer 11 and Edge.
-For a complete breakdown of supported media extensions please contact support@vualto.com
+For a complete breakdown of supported media extensions please contact <support@jwplayer.com>

```

Comment:
Is there not a doc page that we can link to for this? I imagine it is documented in the public docs.